Library/Homebrew/bintray.rb

```diff
@@ -86,34 +95,41 @@ def publish(repo:, package:, version:, file_count:, warn_on_error: false)
     end
 
     odebug "Published #{json["files"]} bottles"
-    result
   end
 
+  sig { params(org: String).returns(T::Boolean) }
   def official_org?(org: @bintray_org)
     %w[homebrew linuxbrew].include? org
   end
 
+  sig { params(url: String).returns(T::Boolean) }
   def stable_mirrored?(url)
     headers, = curl_output("--connect-timeout", "15", "--location", "--head", url)
     status_code = headers.scan(%r{^HTTP/.* (\d+)}).last.first
     status_code.start_with?("2")
   end
 
+  sig do
+    params(formula: Formula, repo: String, publish_package: T::Boolean,
+           warn_on_error: T::Boolean).returns(String)
+  end
   def mirror_formula(formula, repo: "mirror", publish_package: false, warn_on_error: false)
     package = Utils::Bottles::Bintray.package formula.name
 
     create_package(repo: repo, package: package) unless package_exists?(repo: repo, package: package)
 
-    formula.downloader.fetch
+    # TODO: Remove this type hack when we finish annotating formula.rb
+    downloader = T.let(T.unsafe(formula).downloader, AbstractDownloadStrategy)
```

Comment:
We should avoid doing this and instead leave this file as `typed: false`.

Reply:
This may be fixed by https://github.com/Homebrew/brew/pull/9242.

Reply:
Yes, it is. Thanks for that!